_data/statuses.yaml

```diff
@@ -1,3 +1,4 @@
+- Last Call
 - Draft
 - Accepted
```

Comment:
Is "Approved" and "Accepted" the same thing? (Core EIPs only)

Reply:
Sorry, <kbd>W</kbd> and <kbd>T</kbd> are next to each other on the keyboard :-p